Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM rust:1.59 as build
+
+# create a new empty shell project
+RUN USER=root cargo new --bin sol_to_uml
+WORKDIR /sol_to_uml
+
+# copy over your manifests
+COPY ./Cargo.lock ./Cargo.lock
+COPY ./Cargo.toml ./Cargo.toml
+#COPY Cargo.toml ./sol_to_uml/Cargo.toml
+
+# this build step will cache your dependencies
+RUN cargo build --release
+RUN rm ./src/*.rs
+
+# copy your source tree
+COPY ./src ./src
+
+# build for release
+RUN rm ./target/release/deps/sol_to_uml*
```

Comment:
Do we need this?

Reply:
Probably not, also comes from the guide. Docker works fine without it.